server/app/assets/javascripts/main.ts

```diff
@@ -148,6 +148,12 @@ function removeExistingEnumeratorField(event: Event) {
   // submit the input to maintain entity ordering.
   const enumeratorFieldDiv = removeButton.parentElement
   enumeratorFieldDiv.classList.add('hidden')
+  // Why add the hidden class to the input as well?
```

Comment:
I think this Q&A format is confusing, it reads more like you found something odd and don't understand it, rather than trying to explain a deliberate but non-obvious need.

In general comments are clearer if they're just direct about the situation:

EG:

// We must hide the child in addition to the parent because of .......

Reply:
Switched to a more direct comment. I've generally found the Q&A format to be helpful since it allows for describing multiple non-obvious behaviors. In this case, there's only a single behavior, so it's definitely less useful. I'm fine not using it, especially if it's not consistent with the rest of the codebase.

Reply:
I can def see that. Maybe it just needs more pre-ample or something since it's not a norm that I've seen, or perhaps as you mentioned, in the singleton case it doesn't self-document the structure of the comment clearly.

Reply:
Yah. If I stumble upon a case where I think it's the strictly preferred format, I'll definitely call it out at that point. Thanks for the suggestion!